site/source/docs/optimizing/Optimizing-Code.rst

```diff
@@ -33,6 +33,20 @@ In addition to the ``-Ox`` options, there are separate compiler options that can
 
   -  The meanings of the *emcc* optimization flags (``-O1, -O2`` etc.) are similar to *gcc*, *clang*, and other compilers, but also different because optimizing asm.js and WebAssembly includes some additional types of optimizations. The mapping of the *emcc* levels to the LLVM bitcode optimization levels is documented in the reference.
 
+How Emscripten optimizes
+========================
+
+When compiling source files to object files, Emscripten runs clang and the LLVM optimizer. When compiling to wasm object files (the default with wasm), LLVM also compiles into wasm, and optimizes while doing so.
```

Comment:
I think these 2 sentences are redundant.

Reply:
How about replacing them with `Compiling source files to object files works as you'd expect in a native build system that uses clang and LLVM` ?

Reply:
That would work. Could even extend to "Compiling source files to object files works similarly to a native build system: Emscripten runs clang and the LLVM optimizer."